node_modules/col-activities/lib/notifications/weekly.js

```diff
@@ -514,41 +522,75 @@ var UserTotals = function() {
     'pointsReceived': 0
   };
 
+  // Define a method to increment activity counts.
+  var activitiesDictionary = {
+    'get_asset_comment': 'commentsReceived',
+    'get_asset_comment_reply': 'commentsReceived',
+    'get_like': 'likesReceived'
+  };
+
+  that.incrementActivities = getActivitiesIncrementer(that, activitiesDictionary);
+
+  // Define a method to increment point totals.
```

Comment:
@nicolaasmatthijs Not sure I follow. It's creating a dictionary for purposes of defining a method to increment activity counts. What would you like it to say?